client_src/js/lib/modules/printerSettings.js

```diff
@@ -1,1325 +1,1403 @@
-import OctoFarmClient from "../octofarm.js";
+/* eslint-disable quotes */
+import OctoFarmClient from "../octofarm_client.js";
 import UI from "../functions/ui.js";
-import Script from "./scriptCheck.js";
 import Calc from "../functions/calc.js";
+import Script from "./scriptCheck.js";
 
-let currentIndex = 0;
-
-// let controlDropDown = false;
-
-let currentPrinter = null;
+let currentPrinterIndex;
+let printerOnline;
+let currentPrinter;
+let hasErrorNotificationBeenTriggered = false;
+let pageElements;
```

Comment:
global elements to store page information. 
CurrentPrinterIndex = used when the sse event fires again so we don't have to keep checking it. Number
printerOnline = boolean for status check on printers state. This is relevant to the host status. Not the printer status
currentPrinter = Object containing the current printer
hasErrorNotificationBeenTriggered = boolean that get's updated so the UI alert doesn't keep firing for sse event
pageElements = holder for the page elements